test/integration/api.bats

```diff
@@ -423,9 +443,27 @@ function teardown() {
 	skip
 }
 
-# FIXME
 @test "docker search" {
-	skip
+	start_docker 3
+	swarm_manage
+
+	TEMP_NAME=noimage$RANDOM
+	# search image (not exist), the name of images only [a-z0-9-_.] are allowed
+	run docker_swarm search $TEMP_NAME
```

Comment:
I think this is a bit overkill - could you please `docker_swarm search does_not_exist`?